.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
+  }
```

Comment:
Hm, this shouldn't be committed. Did you run into problems with prettier as the formatter?